### ISSUE TITLE
Partial data return prematurely fails (watchQuery)

### ISSUE THREAD

**Issue by gauravtiwari** [apollostack/apollo-client#193]
Hey folks,

Getting following error, when using `returnPartialData` with `watchQuery` (looks like it's trying to get the data from Store, before it's available)

```
store.js:11Caught an exception! Error: Can't find field posts on object [object Object].(…)
```

**Comment by stubailo** [comment]
This is a pretty bad bug! Thanks for the report, we should get it fixed ASAP.

**Comment by stubailo** [comment]
Oops, looks like we have exactly 0 unit tests for `returnPartialData`. That explains why that broke...

**Comment by stubailo** [comment]
This should be fixed in Apollo Client 0.3.5. Please let me know if it isn't!

**Comment by gauravtiwari** [comment]
@stubailo Thanks Sashko. Yeah, me too looked at unit tests and immediately realised that something is wrong with implementation. Great work 👍
